packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,221 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Usage](#usage)
+  - [Working with non-interactive triggers](#working-with-non-interactive-triggers)
+  - [Custom Triggers](#custom-triggers)
+- [Library](#library)
+  - [Bundle Size](#bundle-size)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop         | Type                                                                     | Default     | Description                                                                                                                             | Required |
+| ------------ | ------------------------------------------------------------------------ | ----------- | --------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| content      | `string`                                                                 | `undefined` | Content of the tooltip                                                                                                                  | ✅       |
```

Comment:
Can we have a newline inside the tooltip content?

Reply:
No, tooltips will only contain short supplement text without newlines. if the content is longer that it needs newlines it should not be inside tooltip. 

Here are the content [guidelines](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=175-812084&t=tdn24pTBE7PD3OY2-0) for tooltip: 

<img width="649" alt="image" src="https://github.com/razorpay/blade/assets/35374649/6fc7732d-ae54-4fc1-947d-de38c10429ea">

Reply:
Hoping the designers agree on this. Marking as resolved.